packages/core/src/graph/process-project-graph.ts

```diff
@@ -37,7 +37,8 @@ function visitProject(
   getDependantProjectsForNxProject(
     projectName,
     context.workspace,
-    ({ projectFile }, dependencyName) => {
+    ({ destProjectFile }, dependencyName) => {
+      var projectFile = utils_1.getProjectFileForNxProjectSync(project)
```

Comment:
Change var to const, move this line above line 37 (this does not change, no reason to refind the file for every dep)

Reply:
Also, this should just be `getProjectFileForNxProjectSync(project)`, without the utils_1. You'll need to add an import on line 8,